### ISSUE TITLE
SSL Cert issue

### ISSUE THREAD

**Issue by Twoure** [Twoure/9anime.bundle#1]
This is a know issue with the version of Python Plex uses.  Refer to [Comment_1210565](https://forums.plex.tv/discussion/comment/1210565/#Comment_1210565) for more info.

```py
URLError: <urlopen error [SSL: TLSV1_ALERT_INTERNAL_ERROR] tlsv1 alert internal error (_ssl.c:590)>
```

Will have to either, ask for proxy access, or wait for site to change ssl cert method.

**Comment by Braintelligence** [comment]
Looking forward to a fix. Keep up the good work!

**Comment by Twoure** [comment]
Fixed in `dev` branch https://github.com/Twoure/9anime.bundle/commit/d6ff73ae3a91f759429d81000aeb867167a6cf5a

Will give the channel some _TLC_ this weekend and hopefully push the new release ~~next week~~ when I have time _(sorry)_.

**Comment by Twoure** [comment]
~~I see the site is no longer uses `https` so I can start working on this channel again.~~  I've been busy with https://github.com/Twoure/KissNetwork.bundle/issues/53 lately, but will start to shift my attention to this channel in the coming weeks.

**Comment by Twoure** [comment]
`ssl` issue fixed in https://github.com/Twoure/9anime.bundle/compare/b2e0448...9da443e _(left in `dev` branch for now, but will push as new release soonish)_.

**Comment by Twoure** [comment]
Fixed in [v1.0.1](https://github.com/Twoure/9anime.bundle/releases/tag/v1.0.1).
